google/ggrc-core
# GGRC-4509 Apply proposal doesn't lead to create revision

# Issue description

Apply proposal with changed CAVs to empty doesn't lead to create revisions. 

# Steps to test the changes

1) Create Control/Risk 
2) propose to change CAV from empty to some value (remember it)
3) apply created proposal 
4) propose to change same CAV from value to empty
5) apply that proposal, revision should be created 
6) proposal to change same CAV from empty to remembered value 
7) proposal shouldn't be empty

# Solution description

Update modified_by and updated_at fields if any proposed values will be applied. 

# Sanity checklist




# PR Review checklist
